### PR TITLE
485 Leverage dockerignores to better sanitize slim down our images

### PR DESCRIPTION
**Pull request by kacperpONS**
# Motivation and Context

Added .dockerignore

# What has changed

# How to test?

# Links

https://trello.com/c/JvVs8wuU/485-leverage-docker-ignores-to-better-sanitize-slim-down-our-images-5

# Screenshots (if appropriate):

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.gitignore
+Dockerfile
+.github
+.pytest_cache
+/tests
```
Comment (ryangrundy7) [inline .dockerignore:5]: Should it be `tests/` ?

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+.coverage
+.flake8
+.gitignore
+coverage.xml
```
Comment (AdamHawtin) [inline .dockerignore:9]: The `.pytest_cache`, `.coverage` and `coverage.xml` files should be gitignore'd so not needed here.

**Review by AdamHawtin — DISMISSED** [review]
Some files here are ignored. 

Also, this file seems to be ignoring most of the top level files in the repo, so perhaps this one would benefit from using the "ignore everything except the allowed files" pattern, instead of ignoring what we don't need?

File: .dockerignore (hunk 2 above)
Comment (kacperpONS) [reply .dockerignore:9]: I must've committed these files before adding the .gitignore, I'll remove them from here. 👍

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,11 @@
+.gitignore
+Dockerfile
+.github
+tests/
+.flake8
+.gitignore
```
Comment (AdamHawtin) [inline .dockerignore:6]: duplicate line
```suggestion
```

**Pull request merged**